common/src/main/java/io/netty/util/Recycler.java

```diff
@@ -71,6 +72,11 @@ public void recycle(Object object) {
                 SystemPropertyUtil.getInt("io.netty.recycler.maxSharedCapacityFactor",
                         2));
 
+        MAX_DELAYED_QUEUES = max(0,
+                SystemPropertyUtil.getInt("io.netty.recycler.maxDelayedQueues",
```

Comment:
sure... Do we also want to do a followup and add the same "suffix" to other properties that are per thread ?